xsd/pm/claims.xsd

```diff
@@ -53,7 +53,14 @@ SOFTWARE.
         </xs:complexType>
       </xs:element>
     </xs:all>
-    <xs:attribute name="id" type="xs:long" use="required"/>
+    <!--
+      @todo #388:30min Remove id attribute.
+       Claim id constraint was changed from long to uuid,
+       but to process existing claims with long id attribute after schema upgrade
+       old value was marked as optional to remove it later in next release.
+    -->
+    <xs:attribute name="id" type="xs:long" use="optional"/>
+    <xs:attribute name="uuid" type="uuid" use="optional"/>
```

Comment:
@g4s8 I think it would be easier to migrate, if you leave the field name as `id`, but change the type to string.
And after all the numeric `id` are processed, change this `id` type to uuid.

If we leave it with two optional id and uuid, farm will need to be able to process both types, which will unnecessary complicate it - and then remove it. There is no point in adding that temporary complexity.